software/fork-recipes.yml

```
@@ -0,0 +1,11 @@
+name: "Fork Recipes"
+website_url: "https://mikebgrep.github.io/forkapi/clients/"
+source_code_url: "https://github.com/mikebgrep/fork.recipes"
+description: "Web application that manages your food recipes with simplicity."
+licenses:
+  - BSD-3-Clause
+platforms:
+  - Python
+  - Docker
```

> https://mikebgrep.github.io/forkapi/clients/

Only docker from what I can see
```suggestion
platforms:
  - Docker
```